### DISCUSSION TITLE
Welcome to retro-ipod-spotify-client Discussions!

### DISCUSSION THREAD

**Discussion by dupontgu** [dupontgu/retro-ipod-spotify-client#45] [Announcements]
## 👋 Welcome!
I'm so happy this repo has stayed active since I first posted the project. Grateful for so many of you willing to help each other! It was requested that I open up the Discussions page rather than leaning on the Issues list. Please keep things respectful! I'll hop in every once in a while.

**Comment by nassafellow** [comment]
I'm really stuck.  I just keep getting x set errors.  I've tried this word for word but it isn't working.  Program keeps crashing.  If I do a run through video would you be able to look at it and help me please???

**Reply by doris1347** [reply]
Hello are you still 
stuck?

**Comment by nassafellow** [comment]
Yes still struggling :-(

**Comment by nassafellow** [comment]
I just can't get the UI to come up.

**Reply by doris1347** [reply]
Let me know if you would like help ... or just see issue #69!

**Comment by doris1347** [comment]


**Reply by doris1347** [reply]
Hi Guy, Just figured I drop you a note that an older version of Raspbian (Legacy Buster) and an older version of Raspotify (14.0) solved all my problems!

Thanks again for creating this satisfying project you've made a retired 70 year old very happy! 😃 

-Banks

Check out issue #69 when you have a chance.
